core/prisma/exampleCommunitySeeds/unjournal.ts

```diff
@@ -393,64 +425,108 @@ export default async function main(prisma: PrismaClient, communityUUID: string)
 			id: submissionTypeId,
 			name: "Submission",
 			communityId: communityUUID,
-			fields: {
```

Comment:
I couldn't figure out how to do this connect in prisma now that we've defined our own join table, so went with kysely 🥲

Reply:
you'll never see me complain about replacing prisma with kysely haha